docs/topics/forms/modelforms.txt

```diff
@@ -875,10 +875,13 @@ Providing initial values
 As with regular formsets, it's possible to :ref:`specify initial data
 <formsets-initial-data>` for forms in the formset by specifying an ``initial``
 parameter when instantiating the model formset class returned by
-:func:`~django.forms.models.modelformset_factory`. However, with model
-formsets, the initial values only apply to extra forms, those that aren't
-attached to an existing model instance. If the extra forms with initial data
-aren't changed by the user, they won't be validated or saved.
+:func:`~django.forms.models.modelformset_factory`.
+
```

Comment:
Why was a new paragraph created?

Reply:
I separated it into two paragraphs just because I thought the section was getting a little dense. This is my first documentation PR so I will defer to both of your recommendations in a further commit! Thanks.